contributing/development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -363,6 +363,65 @@ You don't even need to copy them, you can just reference the resulting
 time you build, you automatically have the custom templates referenced.
 
+Cross-compiling for RISC-V devices
+----------------------------------
+
+To cross-compile Godot for RISC-V devices, we need to setup the following items:
+
+- `riscv-gnu-toolchain <https://github.com/riscv-collab/riscv-gnu-toolchain/releases>`__.
+  While we are not going to use this directly, it provides us with a sysroot, as well
+  as header and libraries files that we will need. There are many versions to choose
+  from, however, the older the toolchain, the more compatible our final binaries will be.
+  If in doubt, `use this version <https://github.com/riscv-collab/riscv-gnu-toolchain/releases/tag/2021.12.22>`__,
+  and download ``riscv64-glibc-ubuntu-18.04-nightly-2021.12.22-nightly.tar.gz``. Extract
+  it somewhere and remember its path.
+- Clang. RISC-V GCC has
+  `bugs with its atomic operations <https://github.com/riscv-collab/riscv-gcc/issues/15>`__
+  which prevent it from compiling Godot correctly. Any version of Clang from 16.0.0 upwards
+  will suffice. Download it from the package manager of your distro, and make sure that
+  it *can* compile to RISC-V. You can verify by executing this command ``clang -print-targets``,
+  make sure you see ``riscv64`` on the list of targets.
+- `mold <https://github.com/rui314/mold/releases>`__. This fast linker,
+  is the only one that correctly links the resulting binary. Download it, extract it,
+  and make sure to add its ``bin`` folder to your PATH.
+
+With all the above setup, we are now ready to build Godot.
+
+Go to the root of the source code, and execute the following build command:
+
+::
+
+    scons arch="rv64" use_llvm="yes" linker="mold" lto="none" target="editor" \
+        ccflags="--sysroot=<GCC-TOOLCHAIN-PATH>/sysroot --gcc-toolchain=<GCC-TOOLCHAIN-PATH> -target riscv64-unknown-linux-gnu" \
+        linkflags="--sysroot=<GCC-TOOLCHAIN-PATH>/sysroot --gcc-toolchain=<GCC-TOOLCHAIN-PATH> -target riscv64-unknown-linux-gnu"
+
+The command is similar in nature, but with some key changes. ``ccflags`` and
+``linkflags`` append additional flags to the build. ``--sysroot`` points to
+a folder simulating a Linux system, it contains all the headers, libraries
+and ``.so`` files Clang will use. ``--gcc-toolchain`` tells Clang where
+the complete toolchain is, and ``-target riscv64-unknown-linux-gnu``
+indicates to Clang the target architecture, and OS we want to build for.
+
+Remember to replace ``<GCC-TOOLCHAIN-PATH>`` with the path of the toolchain
+you downloaded!
+
+.. warning:: If you are compiling an older version of Godot, such as 4.2 or 3.x,
+             replace ``ccflags`` with ``CCFLAGS`` and ``linkflags`` with
+             ``LINKFLAGS``.
+
+If all went well, you should now see a ``bin`` directory, and within it,
+a binary similar to the following:
+
+::
+
+    godot.linuxbsd.editor.rv64.llvm
+
+Double-clicking it should bring up the project manager.
```

Comment:
I know this could easily be assumed, but the text should clarify that the executable can only be double-clicked after the executable is copied to the RISC-V machine.

Reply:
Understood. I think a `note` box clarifying it will be enough.

Reply:
Why not just write in the prose? "Copy this executable to a RISC-V machine. Then you can launch it by double-clicking, which should bring up the project manager."

Reply:
Done.